src/main/java/com/aws/greengrass/status/model/ComponentStatusDetails.java

```diff
@@ -17,6 +17,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class ComponentStatusDetails {
-    private List<String> statusCode;
+    private List<String> statusCodes;
```

Comment:
this changes the JSON that this serializes to. Is this backward compatible or is this going to break things?

Reply:
This is intended because cloud is expecting `statusCodes`. Also since this is never released yet, so it won't break anything.